stable/python/_kubeless.py

```diff
@@ -97,18 +97,63 @@ def handler():
                 return res
 
 
+def preload():
```

Comment:
When using `forkserver`, the forkserver is not started until the first process is created. I'm using this function to create that first process before the cherrypy server is even started so that everything is primed and ready to go as soon as the web server is up and we start receiving requests.